modules/oe_webtools_media/src/Plugin/Validation/Constraint/WebtoolsMediaConstraintValidator.php

```diff
@@ -45,6 +49,15 @@ public function validate($value, Constraint $constraint) {
       return;
     }
 
+    // If it's a wcloud style service, assert its correct
+    // and try to fetch the actual snippet for validation.
+    if (!empty($snippet['utility']) && $snippet['utility'] == 'wcloud') {
+      $snippet = $this->parseWcloud($snippet, $constraint);
+      if (empty($snippet)) {
```

Comment:
Imagine that the service, due to an error, returned an empty array 
```
$response = new Response(200, [], '[]');
```
We won't validate that the widget type is correct as we are bailing out.
Let's return FALSE from parseWcloud() when the response is invalid (or an exception).